sema/commons/rdf_ogm/term_builder.py

```
@@ -0,0 +1,73 @@
+        # namespace
+        if self._namespace and (not self._namespace == "@base"):
+            return getattr(Namespace(self._namespace), self._suffix)
+
+        # literal
+        if self._prefix is None and (not self._namespace == "@base"):
+            return Literal(self._suffix)
+        
+        # uriref w/ base
+        if self._namespace == "@base":
+            if not self._graph.base:
+                raise AssertionError("Graph needs base attribute when using base namespace to expand term")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)